Dockerfile.in

```diff
@@ -54,8 +52,6 @@ COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/script
 RUN mkdir -p /usr/share/logging/
 COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/files/ /usr/share/logging/
 
-COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/manifests /manifests
-
```

Comment:
We need to double check this on midstream/downstream. IIRC until 5.1 we used the CLO container image built by brew to extract the manifests and compile the bundle container with brew. I believe we don't do this anymore and we don't need these lines anymore.

Reply:
I put the generated files back into git for CI. I took the COPY config line out of Dockerfile (equivalent of old manifests line), we'll see how that does with CI.